core/src/com/fourinachamber/fortyfive/game/card/Card.kt

```diff
@@ -408,59 +410,81 @@ class Card(
  */
 class CardActor(
     val card: Card,
-    font: BitmapFont,
-    fontColor: Color,
-    fontScale: Float,
-    detailBackgroundHandle: ResourceHandle,
-    detailSpacing: Float,
+    val font: PixmapFont,
+    val fontColor: Color,
+    val fontScale: Float,
     private val screen: OnjScreen
-) : CustomImageActor(card.drawableHandle, screen) {
+) : Widget(), ZIndexActor, HoverStateActor, KeySelectableActor {
 
-    override var isSelected: Boolean = false
+    override var fixedZIndex: Int = 0
 
+    override var isHoveredOver: Boolean = false
+
+    override var isSelected: Boolean = false
     override var partOfHierarchy: Boolean = true
 
     /**
      * true when the card is dragged; set by [CardDragSource][com.fourinachamber.fortyfive.game.card.CardDragSource]
      */
     var isDragged: Boolean = false
 
-    val hoverDetailActor = CardDetailActor(
-        card = card,
-        initialFlavourText = "",
-        initialDescription = "",
-        initialStatsText = "",
-        initialStatsChangedText = "",
-        font = font,
-        fontColor = fontColor,
-        fontScale = fontScale,
-        initialForcedWidth = 0f,
-        backgroundHandle = detailBackgroundHandle,
-        spacing = detailSpacing,
-        screen = screen
-    )
-
     private var inGlowAnim: Boolean = false
+    private var inDestroyAnim: Boolean = false
 
     private val glowShader: BetterShader by lazy {
         ResourceManager.get(screen, "glow_shader") // TODO: fix
     }
+    private val destroyShader: BetterShader by lazy {
+        ResourceManager.get(screen, "dissolve_shader") // TODO: fix
```

Comment:
to push a "TODO: fix" onto main seems kinda counterintuitive, but ik it makes sense sometimes, just wanted to say it, because literally 4 or 5 lines over that is the same